Content.Shared/_RMC14/Stealth/RMCPassiveStealthComponent.cs

```diff
@@ -0,0 +1,23 @@
+using Content.Shared.Whitelist;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared._RMC14.Stealth;
+
+/// <summary>
+///     Will slowly lower an entity's opacity
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class RMCPassiveStealthComponent : Component
+{
+    [DataField, AutoNetworkedField]
+    public float MinOpacity = 0.2f;
```

Comment:
On the video it seemed instant but in 13 it is supposed to slowly turn the entity invisible down to 0, at least for the stealth body bag that the scout gets. Like, actually 0. You can probably do the slowly part by storing a timespan of when this was activated and a delay for full invis, then interpolate the invis amount from that.